src/auth/screens/auth-profile.screen.js

```diff
@@ -81,12 +85,15 @@ class AuthProfile extends Component {
     getUser: Function,
     getOrgs: Function,
     getStarCount: Function,
+    searchUserOpenPullRequests: Function,
     user: Object,
     orgs: Array,
+    searchedUserOpenPullRequests: Array,
```

Comment:
`searchUserOpenPullRequests` and `searchedUserOpenPullRequests` are really easy to be mixed.